lib/upgrade_format_task.rb

```diff
@@ -51,13 +57,12 @@ def upgrade(store, name, options = {})
     end
 
     if options[:all]
+      migrations_done == 1? frequency = "description": frequency = "descriptions"
```

Comment:
Surrounding space missing for operator '?'.<br>Surrounding space missing for operator ':'.